cypress/e2e/0-guild/1-edit-guild.spec.ts

```diff
@@ -0,0 +1,51 @@
+const CONTEXT = {
+  guild: undefined,
+}
+
+const URL_NAME = `${Cypress.env("platformlessGuildUrlName")}-${Cypress.env(
+  "RUN_ID"
+)}`
+
+describe("edit guild", () => {
```

Comment:
this test is not too detailed, I just wanted to test the basic functionality & made a liner issue for a more detailed test ;)